### DISCUSSION TITLE
Ask a question，How to change language of the server?

### DISCUSSION THREAD

**Discussion by viewfirst** [Pryaxis/TShock#2150] [Q&A]
I need to consult with you. How should I change the language of the server? For example, after using a certain command, the system's feedback language is English, and I want to change it to Chinese.

for example，
Enter /reload, the system prompt is "Configuration,permissions,and regions reload complete,some changes may require a server restart ",I hope this system prompt is Chinese

**Comment by bartico6** [comment]
The server doesn't support easy localisation, you have to edit the code to change the responses from the server.

**Reply by viewfirst** [reply]
How can I do it?

**Reply by bartico6** [reply]
You'd have to edit the server code and change the text in .SendSuccessMessage etc to your liking. There is no localisation support at the moment.

**Reply by viewfirst** [reply]
So, do I have to change the language when I create a new server?

**Reply by bartico6** [reply]
No, you have to edit the server code (like, clone tshock and change the code of commands and such)
